packages/fxa-auth-server/lib/routes/utils/signup.js

```diff
@@ -5,6 +5,15 @@
 'use strict';
 const userAgent = require('../../userAgent');
 
+function isSyncBased(service) {
+  return (
+    service === 'sync' ||
```

Comment:
From my understanding, all these clients interact with sync or use sync. Is there a better way to identify that we should send the post verify sync email?

Reply:
I see what you're going for here, but I really hope we can find something better, because otherwise we will be constantly forgetting to add new clients to this list.

Reply:
A more invasive, but perhaps more sustainable option would be to:

* Have `/session/verify_code` accept a `scope` argument in the payload alongside `service`.
* Teach fxa-content-server to pass the requested scopes when verifying the session, the same way it passes `service` right now.

That's more work up-front but would avoid having to maintain this list in future.